package com.etc.Ayy_exception;

public class Exception异常的执行流程 {
    /**
     异常在执行到错误的那一行代码的时候,首先会产生一个异常对象,这个异常对象首先会返回到这个方法的
     调用入口,再返回到main方法再次返回给虚拟机,虚拟机会在控制栏打印出这个错误,然后在异常处杀死
     这个进程,程序后面的就不会执行了.
     在控制栏打印的错误首先会打印出错的地方,然后会打印调用的地方.按照程序的入口.先进后出,所以叫异常堆栈处理


     */
}
